src/yggdrasil/switch.go

```diff
@@ -474,10 +455,7 @@ func (t *switchTable) _handleMsg(msg *switchMsg, fromPort switchPort, reprocessi
 				return false
 			}
 		}
-		if sender.locator.root == t.core.sigPub {
-			return false
-		}
-		return true
+		return sender.locator.root == t.core.sigPub
```

Comment:
It might just be that I haven't slept so I'm tired, but this looks like it returns the wrong value (should be `!=`), though I strongly dislike this change (it makes the logic less clear in a part of the code that's already hard to understand).